<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lesson3</class>
 <widget class="QDialog" name="Lesson3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayoutMain">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Simple Formula</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Full Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEmail"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Birthday:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditBirthDay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>1990</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
             <year>3000</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Preffered Salary:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxSalary">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>90000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>4200</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sex:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButtonSex_1">
           <property name="text">
            <string>Male</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSex_2">
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Personality Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QRadioButton" name="radioButtonPersonality_1">
           <property name="text">
            <string>Extravert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPersonality_2">
           <property name="text">
            <string>Introvert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPersonality_3">
           <property name="text">
            <string>Ambivient</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>Accuracy</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="checkBox_7">
           <property name="text">
            <string>Analytical</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Diligence</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBox_6">
           <property name="text">
            <string>Creative</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>Open-Minded</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>Communicativeness</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QCheckBox" name="checkBox_8">
           <property name="text">
            <string>Responsibility</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxConfident">
           <property name="text">
            <string>Confident</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Select Position:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="checkBox_9">
           <property name="text">
            <string>Teamwork</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="checkBoxAdaptability">
           <property name="text">
            <string>Adaptability</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>Any</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Programmer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Accountant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Recruiter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Manager</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tester</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Receptionist</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Analyst</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Your Strengths (MAX 3)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QDialogButtonBox" name="buttonBoxAcceptReject">
           <property name="standardButtons">
            <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableViewMain"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
